modules/sandboxed-containers-viewing-workloads-from-web-console.adoc

```diff
@@ -3,5 +3,25 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-viewing-workloads-from-web-console_{context}"]
+= Viewing {sandboxed-containers-first} workloads from the web console
 
-= Viewing sandboxed containers workloads from the web console
+{sandboxed-containers-first} based workloads look and feel as normal workloads when viewed in the console. The only difference between the two is the `runtimeClassName`. `runtimeClassName` is what makes a workload run based on different runtimes. In this context, the runtime enabled by {sandboxed-containers-first} is `kata`. You can view the `runtimeClass` that the pods for your workloads use.
+
+.Prerequisites
+
+* You have {product-title} 4.8 installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Navigate to *Administration* -> *Workloads*.
+
+. Identify the type of workload you want to view details for. For example, `Pod`, `Deployment`, `DeploymentConfigs` objects and so on.
+
+. Choose the corresponding workload from the list.
+
+. On the *Details* page, navigate to `runtimeClass`.
+
+.. Hover over `runtimeClass` to view more information.
+.. You can find `kata` as the value of the `runtimeClass`, if `kata` was used as the runtime.
```

Comment:
Suggest flipping this last item around:
> If `kata` was used as the runtime, the value of the `runtimeClass` is `kata`.

That could be added to the line above, since it's more what the user would expect to see than a real step, and "Hover ..." could just become step 5 rather than part of step 4.